pkg/conduit/entrypoint.go

```diff
@@ -163,18 +173,21 @@ func Flags(cfg *Config) *flag.FlagSet {
 
 	flags.BoolVar(&cfg.Preview.PipelineArchV2, "preview.pipeline-arch-v2", cfg.Preview.PipelineArchV2, "enables experimental pipeline architecture v2 (note that the new architecture currently supports only 1 source and 1 destination per pipeline)")
 
-	// NB: flags with prefix dev.* are hidden from help output by default, they only show up using '-dev -help'
-	showDevHelp := flags.Bool("dev", false, "used together with the dev flag it shows dev flags")
```

Comment:
Hmm, shouldn't we keep this? Or do you think we shouldn't provide an option to show these at all?

Reply:
@hariso as far as I can tell, that flag wasn't really doing anything. Going to check with an old version of conduit to see if in some point it should have.

Reply:
@hariso looks like what I was testing was another regression

```
go run ./cmd/conduit/main.go --dev --help
Usage of conduit:
  -dev.blockprofile string
    	write block profile to file
  -dev.cpuprofile string
    	write cpu profile to file
  -dev.memprofile string
    	write memory profile to file
```

That's not working on main now either. Will fix

Reply:
As per standup, we decided to leave these visible.